### ISSUE TITLE
UI for spectrum matcher

### ISSUE THREAD

**Issue by RichCaloggero** [geraintluff/jsfx#4]
I'd like to use spectrum matcher, but as a blind screen reader user, I cannot interact at all with the UI.

THis is strange, because when I looked briefly at the source, there are a bunch of slider definitions at the top of the file. In general, these generate perfectly usable and accessible UI controls which my screen reader can quite happily interact with. SO why are they not showing up here?

Perhaps your ui library is overriding all of them and drawing it's own UI? If this is the case, is there any way to disable tis and use the JSFX standard UI instead?

I heard the demo for this plugin and I'm extremely impressed; would love to be able to try it out!

Thanx for doing what you do!

-- Rich
